static/window/styles/fancy.css

```diff
@@ -760,11 +760,23 @@ body {
 .playlist-list-content__description {
   color: var(--layout__secondary-text-color);
 }
+
+.playlist-list-content__extreme {
+  color: var(--layout__warning-bright-color);
+  font-style: italic;
+}
+
 .playlist-list-content--edit .playlist-list-content__description {
   background-color: var(--layout__playlist-list-item-editing-description-background);
   color: var(--layout__primary-text-color);
 }
-/* Playlist-List-Content Buttons */
+.playlist-list-content--edit .playlist-list-content__extreme {
+  color: var(--layout__primary-text-color);
+  font-style: normal;
+  margin-left: 0.3rem;
```

Comment:
The launcher has two separate CSS files for styling (``fancy.css`` and ``core.css``). This file is used exclusively for stylistic changes (change of color, font, border style etc.) and the other file is for all other changes (margins, width/height, offset etc.)

Therefore ``margin-left`` should be moved to ``core.css`` (at line 1798, since we try to keep the selectors in the same order in both files).

Reply:
moved spacing to core.css